images/php/fpm/entrypoints/51-production-detection.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [[ ${LAGOON_ENVIRONMENT_TYPE} == "production" && ( -z "${production_notices}" || $production_notices == "true" ) ]]; then
```

Comment:
what is `production_notices` and how is it defined?

Reply:
also can we change the script so that if somebody wants to overwrite `PHP_ERROR_REPORTING` from the outside via an environment variable that this is possible? Currently it's not possible to set it, as it will be set by the entry point script all the time

Reply:
`production_notices` is just an env var that can be defined to enable notices on a production environment. 

I can add a customer supplied `PHP_ERROR_REPORTING` variable as well, sure.

Reply:
Added logic to accept a customer supplied value for `PHP_ERROR_REPORTING`. If that value is not set, then we set the value automatically based on production/development status.